io.github.josephmawa.Bella.json

```diff
@@ -0,0 +1,57 @@
+{
+  "app-id": "io.github.josephmawa.Bella",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "io.github.josephmawa.Bella",
+  "finish-args": [
+    "--share=network",
```

Comment:
why does this need network?

Reply:
This is my first Flatpak project.  I'm not yet sure what those commands are for.  It is contained in the project template that  GNOME builder provides. I will remove it since the app seems to work fine without it. Hopefully, it won't break things on other systems.